getting_started/setup_linux/install_locally.rst

```diff
@@ -0,0 +1,106 @@
+.. _install-locally:
+
+Install the Kernel locally
+#######################################
+
+.. contents::
+   :local:
+   :depth: 3
+
+Introduction
+************
+
+Make sure you have `setup your development environment <prepare_build_environment.html>`_ before following these steps. This page will guide you through the process of installing the kernel locally on your machine. It will be installed in addition to your distro's default kernel so that you can always change back to that in case something goes wrong.
+
+
+Build and install the kernel
+****************************
+
+(You should be in the ``~/sof/linux`` dir you created on the setup page)
+
+1. Load base kernel configuration
+---------------------------------
+
+.. code-block:: bash
+
+   # This will copy the booted kernel's configuration so that it will be used as a base
+   cp /boot/config-`uname -r`* .config
+
+
+2. Apply SOF-specific configuration
+-----------------------------------
+
+The following scripts will update your base config so that it uses the latest SOF modules. Run only one of them depending on your needs. If it prompts you with any questions, just press <enter> to accept the default value. Note that, by default, these scripts will set the configuration to only compile modules that are currently loaded in order to lower compile times. This means that when you've booted from the custom kernel some external devices may not work if they weren't connected while running this script. If you want to compile all modules, delete the line ``make localmodconfig`` from the script you will run in this step.
+
+.. code-block:: bash
+
+   # For most users
+   ../kconfig/kconfig-distro-sof-update.sh
+   # For additional logging and experimental device support
+   ../kconfig/kconfig-distro-sof-dev-update.sh
+
+3. Compile the kernel
+---------------------
+
+.. code-block:: bash
+
+   # This can take a while (over 30 minutes on some machines),
+   # so grab a coffee or take an exercise break while it runs
+   make -j`nproc --all`
+
+4. Install the kernel
+---------------------
+
+.. code-block:: bash
+
+   sudo make modules_install
+   sudo make install
+
+If all went well, your freshly-built kernel will be installed and available at next boot. Restart your computer, and you should have the option to pick a kernel when it turns on. Select the kernel that has "-sof" at the end of it, and your computer should boot as normal using the kernel you just built. On Ubuntu, the kernel option may be hidden behind the "Advanced options for Ubuntu" submenu.
```

Comment:
Why did you remove the GRUB_DISABLE_SUBMENU advice above?

Reply:
that's only needed for ktest @marc-hb 

But to your point we could add something that says that to see all options the submenu can be disabled.